### PR TITLE
Feed card updates

### PR DESCRIPTION
**Pull request by asleepingpanda**

### CODE REVIEW

**Comment by asleepingpanda** [comment]
This PR adds an additional field to the HorizontalCardListFeature so that we can determine the style of card that we want showing up in the feed

File: src/data/event-content-item/resolver.js [hunk 1]
```diff
@@ -49,25 +49,8 @@ const resolver = {
...
-
-      // Sort by start date asc, take the first
-      const eventStart = first(schedules.sort((a, b) => moment(a.effectiveStartDate).diff(b.effectiveStartDate)))
-
-      // Sort by end date desc, take the first
-      const eventEnd = first(schedules.sort((a, b) => moment(b.effectiveEndDate).diff(a.effectiveEndDate)))
-
-      return ""
-    },
+    label: async ({ attributeValues }) =>
```
Comment (jordanwade) [inline src/data/event-content-item/resolver.js:52]: Is this change a result of dead/deprecated code or something else?

File: src/data/features/schema.js [hunk 2]
```diff
@@ -11,8 +11,16 @@ export default gql`
         READ_GROUP
     }
 
+    enum HORIZONTAL_CARD_TYPE {
```
Comment (rkdavidson) [inline src/data/features/schema.js:14]: Aren't `enums` typically in `TitleCase` too? Then the values in `CAPS_SNAKE_CASE`?

```suggestion
    enum HorizontalCardType {
```

**Review by rkdavidson — COMMENTED** [review]
Left a code naming/styling comment but I think these changes look good... haven't tested though

File: src/data/event-content-item/resolver.js (hunk 1 above)
Comment (asleepingpanda) [reply src/data/event-content-item/resolver.js:52]: Yeah I snuck that in there, too. Just updating the way that Labels get pulled in from Rock. Simplified it quite a bit.

File: src/data/features/schema.js (hunk 2 above)
Comment (asleepingpanda) [reply src/data/features/schema.js:14]: Gotcha! Sounds good to me. I see both conventions in the Core Apollos project, but if this approach is more typical of GQL globally, let's for sure go with Upper Camel Case

**Pull request merged**